### ISSUE TITLE
Problem in iOS 

### ISSUE THREAD

**Issue by alirezaimi** [seriyps/mtproto_proxy#67]
Hi
I implemented erlang mtproto on my server and everything is fine on Android devices and desktop users, but on iOS device there is a bad connection issue that annoy me.
Why is this happen ???

**Comment by seriyps** [comment]
Hi. Are you trying to connect from Iran? Using FakeTLS? Might be that packet format iOS uses is detected by firewall. But it's the 1st tie I hear about this problem

**Comment by alirezaimi** [comment]
yes, i'm trying from iran, sadly on iphones with faketls this problem is so painful .
The telegram on ios device trying for many times and suddenly connected with very slow connection speed .

**Comment by shanoaice** [comment]
It seemed like that using the IP address of the MTProto Proxy will solve the problem. Unsure why.

**Comment by alirezaimi** [comment]
@shanoaice I use only ip for this and tested on iphone 13 with ios 15 and the problem still there! :(

**Comment by Odinman** [comment]
same problem
